### DISCUSSION TITLE
Add a keymap option with all the ascii characters on the keyboard similar to keybr's layout to help practicing with other symbols.

### DISCUSSION THREAD

**Discussion by gordonn** [monkeytypegame/monkeytype#5351] [Ideas]
![Screenshot 2024-05-01 130642](https://github.com/monkeytypegame/monkeytype/assets/121423020/35103cf3-5e3b-4e73-b355-6c2a470e5f95)

**Comment by gordonn** [comment]
I have found out the solution, thanks to a fellow community member: 

[1] Under (Settings), [Keymap Show Top Row] choose ((Always)).
[The top row of numbers will 'always' be included with the keymap.]

[2] Under [Keymap Legend Style] choose ((Dynamic)).
[The keymap will display the upper row as numbers unless you hold down a (shift) key; then the whole keyboard will show the (upper/shifted) letters and symbols. "Dynamic" will change the look (lowercase, uppercase, number, symbol).]
★[You have to have "punctuation" and "numbers" turned (on) for those keys to be necessary when typing.]

[3] Under (Settings), go down to [Keymap] and turn on ((Next)).
[This will show you the NEXT key you are supposed to hit.]

**Reply by Ottaviocr** [reply]
Thanks for this. I was going to ask the same question.
